src/components/pages/AdminTools/AddFaq.js

```diff
@@ -77,51 +57,65 @@ const AddFaq = props => {
+    setIsModalVisible(false);
+    // history.push('/manage-faq');
   };
 
   return (
-    <div className={classes.root}>
-      <div className={classes.form}>
-        <h2 className={classes.h1Styles}> Add Question </h2>
-        <form onSubmit={onSubmit}>
-          <label htmlFor="question">
-            <TextField
-              id="question"
-              label="Question"
-              type="text"
-              name="question"
-              variant="outlined"
-              multiline={true}
-              onChange={onChange}
-              className={classes.textField}
-              value={formValues.question}
-            />
-          </label>
-          <label htmlFor="answer">
-            <TextField
-              id="answer"
-              label="Answer"
-              type="text"
-              name="answer"
-              variant="outlined"
-              multiline={true}
-              onChange={onChange}
-              className={classes.textField}
-              value={formValues.answer}
-            />
-          </label>
-          <div className="submit-button">
-            <Button
-              onClick={onSubmit}
-              className={classes.buttonStyles}
-              variant="contained"
-              component="span"
-            >
-              Submit
-            </Button>
-          </div>
-        </form>
+    <div className="root">
+      <div className="add-faq-btn-container">
+        <Button className="add-faq-btn" onClick={showModal}>
+          Add a FAQ
+        </Button>
+        <Modal
+          title=""
+          visible={isModalVisible}
+          onOk={handleOk}
+          onCancel={handleCancel}
+          footer={[
+            <div className="submit-button">
+              <Button
+                htmlType="submit"
+                className="add-faq-btn"
+                onClick={onSubmit}
+              >
+                <span>Submit</span>
+              </Button>
+            </div>,
+          ]}
+        >
+          <Form layout="vertical" className="faq-form" onFinish={onSubmit}>
+            <h2 className="h1Styles">Add a FAQ</h2>
+            <p className="divider">
+              <Icon
+                component={() => <img src={OrangeLine} alt="divider icon" />}
+              />
+            </p>
+            <Form.Item label="Question">
+              <Input
+                id="question"
+                type="text"
+                name="question"
+                // autoSize={{ minRows: 2, maxRows: 8 }}
+                onChange={onChange}
+                className="text-field"
+                value={formValues.question}
+              />
+            </Form.Item>
+            <Form.Item label="Answer">
+              <Input
+                id="answer"
+                type="text"
+                name="answer"
+                // autoSize={{ minRows: 4, maxRows: 10 }}
+                onChange={onChange}
+                // onValuesChange={onChange}

```

Comment:
Just a few lines that can be commented out. I guess this code stayed during the transfer!! Aside from that, I'll go ahead and approve this. Excited for the styling to apply consistently across the admin pages!

Reply:
I will get these cleaned up now and push a new commit!

Reply:
Beautiful!